[Display_01] Define DisplayGpsReceiverImplementationTests

EC01EC opened the pull request:
Created test methods for DisplayGpsReceiverImplementationTests.puml

Fixes for issues #56 and #74

EC01EC committed:
Update DisplayGpsReceiverImplementationTests.puml

Created test methods for DisplayGpsReceiverImplementationTests.puml

EC01EC committed:
Merge branch 'main' into define-DisplayGpsReceiverTests

EC01EC committed:
Generate SVG and PNG Images for PlantUML Diagrams

EC01EC committed:
Update DisplayGpsReceiverImplementationTests.puml

Updated 3rd method from "if" to "else"

EC01EC committed:
Generate SVG and PNG Images for PlantUML Diagrams

EC01EC committed:
Updates to DisplayGpsReceiver test and production files

Changed passed data type from 'String satelliteName' to 'Satellite satelliteSignal' in .puml diagrams for test classes and in production code

EC01EC committed:
Merge branch 'define-DisplayGpsReceiverTests' of https://github.com/andrewdimmer/csi-3370-software-project into define-DisplayGpsReceiverTests

EC01EC committed:
Generate SVG and PNG Images for PlantUML Diagrams

andrewdimmer commented on docs/uml-class-diagrams/display01/test/DisplayGpsReceiverImplementationTests/DisplayGpsReceiverImplementationTests.puml:
```diff
@@ -2,7 +2,9 @@
-
+        + void measureGpsSignalStrength()
+        + void ifSignalStrengthGoodMeasureSignal()
+        + void elseSignalStrengthBadRejectSignal()
```
Make sure you have a check that the next link in the chain is not missing!
```suggestion
        + void elseSignalStrengthBadRejectSignal()
        + void displayGpsManagerIsNotNull()
```

andrewdimmer reviewed (changes requested):
One minor change, then you're set to merge!

EC01EC committed:
Update docs/uml-class-diagrams/display01/test/DisplayGpsReceiverImplementationTests/DisplayGpsReceiverImplementationTests.puml

Co-authored-by: Andrew Dimmer <43498863+andrewdimmer@users.noreply.github.com>

EC01EC committed:
Generate SVG and PNG Images for PlantUML Diagrams

EC01EC committed:
Update DisplayGpsReceiverImplementationTests.puml

Removed 'if/else' checks after checks of UC diagrams indicated that they were unnecessary.

andrewdimmer reviewed (approved):
Looks good to me, you're all set to merge!

EC01EC committed:
Generate SVG and PNG Images for PlantUML Diagrams

EC01EC committed:
Merge branch 'main' into define-DisplayGpsReceiverTests

EC01EC committed:
Generate SVG and PNG Images for PlantUML Diagrams

stevecatherman reviewed (approved):
:)

andrewdimmer committed:
Merge branch 'main' into define-DisplayGpsReceiverTests

andrewdimmer committed:
Generate SVG and PNG Images for PlantUML Diagrams
